iso15118/secc/states/iso15118_2_states.py

```diff
@@ -2360,7 +2328,7 @@ async def process_message(
         # Because there are EVs that send a wrong Precharge-Voltage
         # in the first message (example: BMW i3 Rex 2018)
         await self.comm_session.evse_controller.set_precharge(
-            precharge_req.ev_target_voltage, precharge_req.ev_target_current
+            ev_data_context.target_current, ev_data_context.target_voltage
```

Comment:
set_precharge() expects PVEVTargetVoltage, PVEVTargetCurrent types.
```suggestion
            precharge_req.ev_target_voltage, precharge_req.ev_target_current
```

Reply:
here the intention is actually to use the data in the EV data context and not directly from the PrechargeReq message

Reply:
it was updated, closing for a fresh review